packages/terra-form-select/src/search/Dropdown.module.scss

```diff
@@ -0,0 +1,22 @@
+:local {
+  .dropdown {
+    background-color: var(--terra-form-select-dropdown-background-color, #fff);
+    border: var(--terra-form-select-dropdown-border, 1px solid #dedfe0);
+    border-bottom-left-radius: var(--terra-form-select-dropdown-border-bottom-left-radius, 3px);
+    border-bottom-right-radius: var(--terra-form-select-dropdown-border-bottom-right-radius, 3px);
+    border-top: 0;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    box-shadow: var(--terra-form-select-dropdown-box-shadow, 0 3px 12px -5px rgba(0, 0, 0, 0.25));
+  }
```

Comment:
Things start to get a little dicey with theme-able variables. Will each variant redefine the dropdown scss or is this an intermediate step while uplifting all variants?

Reply:
This is just an intermediate step.

Reply:
The only change here from the one in `src/shared/` is that this removes the `overflow: auto;`